### PR TITLE
Add Google

### PR DESCRIPTION
**Pull request by jamesward**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ A huge thank you to our sponsors:
 * [Apache SkyWalking](http://github.com/apache/skywalking) - End-to-end testing of the Apache SkyWalking, and plugin tests of its subproject, [Apache SkyWalking Python](http://github.com/apache/skywalking-python), and of its eco-system built by the community, like [SkyAPM NodeJS Agent](http://github.com/SkyAPM/nodejs)
 * [jOOQ](https://www.jooq.org) - Integration testing all of jOOQ with a variety of RDBMS
 * [Trino (formerly Presto SQL)](https://trino.io) - Integration testing all Trino core & connectors, including tests of multi-node deployments and security configurations.
+* [Google]() - Various open source projects: [OpenTelemetry](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java), [Universal Application Tool](https://github.com/seattle-uat/universal-application-tool), [CloudBowl](https://github.com/GoogleCloudPlatform/cloudbowl-microservice-game)
```
Comment (rnorth) [inline docs/index.md:166]: Maybe `Google` should be linked to google.com?
```suggestion
* [Google](https://google.com) - Various open source projects: [OpenTelemetry](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java), [Universal Application Tool](https://github.com/seattle-uat/universal-application-tool), [CloudBowl](https://github.com/GoogleCloudPlatform/cloudbowl-microservice-game)
```

Or not linked at all?

```suggestion
* Google - Various open source projects: [OpenTelemetry](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java), [Universal Application Tool](https://github.com/seattle-uat/universal-application-tool), [CloudBowl](https://github.com/GoogleCloudPlatform/cloudbowl-microservice-game)
```

**Review by rnorth — COMMENTED** [review]
Hi @jamesward - thanks! Obviously we're thrilled to be able to add this to the list 😄 

We just have a little query/comment. I'd be grateful if you could choose whichever suggestion you're happy with, then we can merge 😁

File: docs/index.md (hunk 1 above)
Comment (jamesward) [reply docs/index.md:166]: Ha. :) Whoops.  Forgot to finish that.

**Review by rnorth — APPROVED** [review]

**Comment by rnorth** [comment]
Merged! Thanks @jamesward!

**Pull request merged**